docs/Makefile

```diff
@@ -175,3 +175,12 @@ pseudoxml:
 	$(SPHINXBUILD) -b pseudoxml $(ALLSPHINXOPTS) $(BUILDDIR)/pseudoxml
 	@echo
 	@echo "Build finished. The pseudo-XML files are in $(BUILDDIR)/pseudoxml."
+
+makedocsmessages: gettext
+	cd docs && sphinx-intl update -p _build/locale -l en
+
+ensurecrowdinclient:
+	ls -l crowdin-cli.jar || wget https://crowdin.com/downloads/crowdin-cli.jar # make sure we have the official crowdin cli client
```

Comment:
Have you tried the Python client?

https://pypi.python.org/pypi/crowdin-client